### PR TITLE
[INLONG-962][Release] Add blog for the 1.13.0 release

### LINKED ISSUE #962: [Release] Add blog for the 1.13.0 release
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
Fixes #962

### Motivation

Add blog for the 1.13.0 release.

### Modifications

Add blog for the 1.13.0 release.

### CODE REVIEW

**Review by aloyszhang — APPROVED** [review]

File: blog/2024-07-18-release-1.13.0.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
+title: Release 1.13.0
+author: Wake Fu
```
Comment (aloyszhang) [inline blog/2024-07-18-release-1.13.0.md:3]: who is 'Wake Fu' ?

File: i18n/zh-CN/docusaurus-plugin-content-blog/2024-07-18-release-1.13.0.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
+---
+title: 1.13.0 版本发布
+author: Wake Fu
```
Comment (aloyszhang) [inline i18n/zh-CN/docusaurus-plugin-content-blog/2024-07-18-release-1.13.0.md:3]: ditto

File: blog/2024-07-18-release-1.13.0.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+---
+title: Release 1.13.0
+author: Wenkai Fu
+author_url: https://github.com/fuweng11
+author_image_url: https://avatars.githubusercontent.com/u/8108604?s=400&v=4
```
Comment (aloyszhang) [inline blog/2024-07-18-release-1.13.0.md:5]: wrong  url

File: i18n/zh-CN/docusaurus-plugin-content-blog/2024-07-18-release-1.13.0.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
+---
+title: 1.13.0 版本发布
+author: Wenkai Fu
+author_url: https://github.com/fuweng11
+author_image_url: https://avatars.githubusercontent.com/u/8108604?s=400&v=4
```
Comment (aloyszhang) [inline i18n/zh-CN/docusaurus-plugin-content-blog/2024-07-18-release-1.13.0.md:5]: ditto

File: i18n/zh-CN/docusaurus-plugin-content-blog/2024-07-18-release-1.13.0.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+感谢 @haifxu、@fuweng11 两位同学在 Dashboard 及 Manager 部分对此功能的贡献。具体可参考：INLONG-10330。
+![1.13.0-create-template.png](img%2F1.13.0-create-template.png)
+![1.13.0-select-template.png](img%2F1.13.0-select-template.png)
+![1.13.0-import-template.png](img%2F1.13.0-import-template.png)
+
+### Inlong 支持配置离线同步任务 Pulsar > MySQL
+在1.13.0版本中，InLong 支持了离线同步任务的配置，与实时同步相比，离线数据同步（尚不支持）更注重同步吞吐量和效率。
+该实现统一基于Flink计算引擎。实时同步任务以Flink流任务的方式运行，而离线同步则以Flink批处理任务的方式进行。这种方法可以尽可能地确保实时和离线同步任务代码的一致性，从而降低维护成本。
+InLong 的离线同步功能将与调度系统相结合，将数据源信息的完整或增量数据同步到数据目标，离线同步任务由 InLong Manager 创建（包括调度信息），具体的数据同步逻辑通过 InLong Sort模块实现。
+![1.13.0-manager-offline.png](img%2F1.13.0-manager-offline.png)
```
Comment (aloyszhang) [inline i18n/zh-CN/docusaurus-plugin-content-blog/2024-07-18-release-1.13.0.md:97]: link is broken

File: blog/2024-07-18-release-1.13.0.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+! [1.13.0-select-template.png](img%2F1.13.0-select-template.png)
+! [1.13.0-import-template.png](img%2F1.13.0-import-template.png)
+
+### InLong supports configuring offline synchronization tasks Pulsar > MySQL
+In version 1.13.0, Manager supports the configuration of offline synchronization tasks. Compared to real-time synchronization, offline data synchronization(not supported yet) pays more attention to synchronization throughput and efficiency.
+Real-time synchronization tasks run in the manner of Flink stream tasks, while offline synchronization runs in the manner of Flink batch tasks. This approach can ensure the consistency of real-time and offline synchronization tasks' code as much as possible, reducing maintenance costs.
+The offline synchronization function of InLong will be combined with the scheduling system to synchronize the complete or incremental data of the data source information to the data target. The offline synchronization task is created by InLong Manager (including scheduling information), and the specific data synchronization logic is implemented through the InLong Sort module.
+![1.13.0-manager-offline.png](img%2F1.13.0-manager-offline.png)
```
Comment (aloyszhang) [inline blog/2024-07-18-release-1.13.0.md:93]: link is broken

File: i18n/zh-CN/docusaurus-plugin-content-blog/2024-07-18-release-1.13.0.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+- 特殊处理：脏数据处理能力
+下图为核心流程：
+![1.13.0-dataflow-architecture.png](img%2F1.13.0-dataflow-architecture.png)
+
+### 优化 Sort Standalone 配置流程
+在1.13.0版本中，修改了 Sort Standalone 配置下发流程，在之前的版本中，sort standalone 的配置下发存在以下问题:
+- 配置变更具有不可靠性。配置变化会实时更新到 manager 的缓存中，一旦配置变更，Sort Standalone 就会感知到，并根据新的配置写入数据，并没有进行对配置的校验。
+- 配置构建流程重复并且繁琐。从数据库中拉取 Sort 配置时是全量拉取并且在拉取完毕后进行实时构建。
+在新的版本中，修改数据目标后，Sort 配置将不会实时生效，而是需要在执行工作流后将 Sort 配置构建写入 sort_config 表中。下图为流程对比:
+![1.13.0-dataflow-architecture.png](img%2F1.13.0-dataflow-architecture.png)
```
Comment (aloyszhang) [inline i18n/zh-CN/docusaurus-plugin-content-blog/2024-07-18-release-1.13.0.md:116]: picture content is wrong

**Review by aloyszhang — CHANGES_REQUESTED** [review]
left few comments, PTAL

File: blog/2024-07-18-release-1.13.0.md [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+  The following is the core process:
+![1.13.0-dataflow-architecture.png](img%2F1.13.0-dataflow-architecture.png)
+
+### Optimize the Sort Standalone configuration process
+In version 1.13.0, the distribution process of Sort Standalone configuration was modified. In previous versions, there were the following issues with the distribution of Sort Standalone configuration:
+- Configuration changes are unreliable. Configuration changes will be updated in real-time to the manager's cache. Once the configuration is changed, Sort Standalone will sense it and write data based on the new configuration without verifying the configuration.
+- The configuration and construction process is repetitive and cumbersome. When pulling the Sort configuration from the database, it is a full pull and real-time build is performed after the pull is complete.
+In the new version, after modifying the data target, the Sort configuration will not take effect in real time, but will need to be built and written into the sort_config table after executing the workflow. The following figure shows a process comparison:
+![1.13.0-dataflow-architecture.png](img%2F1.13.0-dataflow-architecture.png)
```
[the remark this answers is not in the document]
Comment (fuweng11) [reply blog/2024-07-18-release-1.13.0.md]: Fixed.

File: i18n/zh-CN/docusaurus-plugin-content-blog/2024-07-18-release-1.13.0.md (hunk 7 above)
Comment (fuweng11) [reply i18n/zh-CN/docusaurus-plugin-content-blog/2024-07-18-release-1.13.0.md]: Fixed.

File: blog/2024-07-18-release-1.13.0.md (hunk 6 above)
Comment (fuweng11) [reply blog/2024-07-18-release-1.13.0.md]: Fixed.

File: i18n/zh-CN/docusaurus-plugin-content-blog/2024-07-18-release-1.13.0.md (hunk 5 above)
Comment (fuweng11) [reply i18n/zh-CN/docusaurus-plugin-content-blog/2024-07-18-release-1.13.0.md]: Fixed.

File: blog/2024-07-18-release-1.13.0.md (hunk 1 above)
Comment (fuweng11) [reply blog/2024-07-18-release-1.13.0.md]: Fixed.

File: blog/2024-07-18-release-1.13.0.md (hunk 3 above)
Comment (fuweng11) [reply blog/2024-07-18-release-1.13.0.md]: Fixed.

File: i18n/zh-CN/docusaurus-plugin-content-blog/2024-07-18-release-1.13.0.md (hunk 4 above)
Comment (fuweng11) [reply i18n/zh-CN/docusaurus-plugin-content-blog/2024-07-18-release-1.13.0.md]: Fixed.

**Review by aloyszhang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**